tests/virt_autotest/hotplugging_utils.pm

```diff
@@ -83,11 +82,9 @@ sub reset_guest {
     script_run("rm -f $disk_image");
     ## CPU and memory
     set_vcpus($guest, 2);
-    my $memory = $guest_instance->{memory} // "2048";
-    set_guest_memory($guest, $memory);
+    set_guest_memory($guest, 2048);
     # max memory
-    my $maxmemory = $guest_instance->{maxmemory} // "4096";
-    script_run("virsh setmaxmem $_ $maxmemory" . "M --config") foreach (keys %virt_autotest::common::guests);
+    script_run("virsh setmaxmem $guest 4096" . "M --config");
```

Comment:
Could you help use variables for memory and maxmemory instead of constants ?

Reply:
This is temporary workaround for a bug so it does not mess up my verification run. This code will be changed back before merge.